patches_for_WebRTC_org/m84/src/modules/video_capture/windows/video_capture_winrt.cc

```diff
@@ -513,15 +513,11 @@ HRESULT VideoCaptureWinRTInternal::FrameArrived(
       hr = bitmap_buffer->GetPlaneCount(&plane_count);
     }
 
-    if (SUCCEEDED(hr)) {
-      hr = plane_count == 2 ? S_OK : E_FAIL;
-    }
-
-    if (SUCCEEDED(hr)) {
+    if (SUCCEEDED(hr) && plane_count >= 1) {
       hr = bitmap_buffer->GetPlaneDescription(0, &bitmap_plane_description_y);
     }
 
-    if (SUCCEEDED(hr)) {
+    if (SUCCEEDED(hr) && plane_count == 2) {
```

Comment:
Previously, plane_count == 3 would have set hr to E_FAIL. Is there a guarantee that plane_count is never any value other than 0, 1, or 2?

Reply:
You're right. It probably makes sense to fail if `plane_count` isn't 1 or 2.